addons/disable-paste-offset/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Do not shift pasted items",
-  "description": "Paste copied items at their original position instead of shifted slightly in the costume editor.",
+  "name": "Unshifted pasted items",
```

Comment:
Old name is better; the addon does not "unshift", it just prevents it from shifting. (Same way people don't say "uneaten cake".)

Reply:
How about "No shift" haha